go/src/plugins/plugins_files/GenericPluginDetection.go

```diff
@@ -0,0 +1,58 @@
+// Example Plugin for reference
+
+package plugins_files
+
+import (
+	"go/src/lib/lomipc"
+	"go/src/plugins/plugins_common"
+)
+
+type GenericPluginDetection struct {
+	plugins_common.PluginMetadata
+	// ... other Internal plugin data
+}
+
+func (gpl *GenericPluginDetection) Request(hbchan chan plugins_common.PluginHeartBeat, request *lomipc.ActionRequestData) *lomipc.ActionResponseData {
```

Comment:
Could you add error as well ?

Reply:
Yes. added it. Please update your plugin LLD.